### ISSUE TITLE
`jupyterlab_bokeh` labextension doesn't install with jupyterlab 0.28

### ISSUE THREAD

**Issue by achimgaedke** [bokeh/jupyterlab_bokeh#8]
I've installed `jupyterlab` via `conda`
```
conda list jupyter
# packages in environment at /home/achim/miniconda3/envs/good-life-adventures:
#
jupyter_client            5.1.0                    py36_0    conda-forge
jupyter_core              4.3.0                    py36_0    conda-forge
jupyterlab                0.28.0                   py36_0    conda-forge
jupyterlab_launcher       0.5.4                    py36_0    conda-forge
```

And the latest `bokeh` from pip
```
pip install bokeh
```

Trying to install the extension with `jupyter labextension install jupyterlab_bokeh` results in:
```
ValueError: 
"jupyterlab_bokeh@0.0.1" is not compatible with the current JupyterLab
Conflicting Dependencies:
JupyterLab              Extension            Package
>=0.11.1-0 <0.12.0-0    >=0.10.0-0 <0.11.0-0 @jupyterlab/application
```

Same, if I use the jupyterlab version from pip. (have read https://github.com/bokeh/jupyterlab_bokeh/issues/4 - and hope/believe it is not a duplicate)

I guess the extension needs an update?! Thanks!

**Comment by ktong** [comment]
PR #6 solves this issue

**Comment by achimgaedke** [comment]
All right, thanks. Tried it out successfully.

(`npm` can be confusing if there is a conda and a ubuntu version of `nodejs` on the system, more confusing as it is by default.)

**Comment by canavandl** [comment]
resolved by #6
